docs/development/UX/strategy.md

```diff
@@ -0,0 +1,48 @@
+# UX strategy
+
```

Comment:
In addition, I propose to add a data transfer function to K when replacing the wallet itself. I think that such an opportunity can increase the rating of the device.

Reply:
see suggestion below